packages/web/src/utils.test.ts

```diff
@@ -0,0 +1,56 @@
+import { parseProperties } from './utils';
+
+describe('utils', () => {
+  describe('parse properties, strip (production)', () => {
+    it('all allowed', () => {
+      const properties = {
+        all: true,
+        should: 'pass',
+        are: true,
+        is: null,
```

Comment:
Could we have a better testing object? Maybe `string: 'some-string', boolean: false`